### DISCUSSION TITLE
Self-messages with keywords can no longer be highlighted

### DISCUSSION THREAD

**Discussion by fowwo** [night/betterttv#5878] [Ideas]
Previously (<7.5.0), messages containing keywords would be highlighted even if the message was sent by oneself. Using my own username as the keyword filter (with the type `Username`), I could highlight my own messages to make them easily discernible. This was useful for finding my own messages in both live chat and in VODs.

However, in the recent release, self-messages can no longer be highlighted.
https://github.com/night/betterttv/blob/89451baf3e6e3d0004943e412eb40f8ac0d4d218/src/modules/chat_highlight_blacklist_keywords/index.js#L142-L146
Is there any chance we can get this "feature" again? I'm not sure if this was implemented to fix another issue, but allowing self-messages to be highlighted seems to be as easy as removing the relevant lines above.

**Comment by night** [comment]
Thanks for the suggestion. This was an unintended change and was fixed in https://github.com/night/betterttv/commit/910f2b90fa7dc3b69ccf491cc794fee90b059cc4
